atd-vzd/schema/atd_txdot_crashes.sql

```diff
@@ -20,15 +18,14 @@ SET row_security = off;
 
 SET default_tablespace = '';
 
-SET default_with_oids = false;
+SET default_table_access_method = heap;
 
 --
--- TOC entry 219 (class 1259 OID 2345633)
--- Name: atd_txdot_crashes; Type: TABLE; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes; Type: TABLE; Schema: public; Owner: visionzero
 --
 
 CREATE TABLE public.atd_txdot_crashes (
```

Comment:
I'd start each of these table definitions right here on the `CREATE`.

Reply:
Rose and I got to sync a few moments ago, and she explained to me that the form of this file is actually matching the other files in this folder, and we looked at it, and she helped me realize that all of the stuff above the CREATE is actually fine, and we agreed that `pg_dump` knows what it's doing. I think my above suggestion isn't helpful should be struck. Thank you @roseeichelmann!